### PR TITLE
Display the owner of the domain on the domain card

### PR DESCRIPTION
**Pull request by Fox32**
image

The changes to `ItemCard` are breaking, so we might want to avoid them. We could also stay with the name `type`. I added the ability to style the EntityRefLinks as a link with the primary color (blue) is hard to read on the green background.

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)

### CODE REVIEW

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4571-bmn58uma9t6a7kwytwzrj7lfalydpfqe.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/602e3d8dfcb22d554558face

File: .changeset/popular-nails-agree.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/core': patch
```
Comment (freben) [inline .changeset/popular-nails-agree.md:2]: so a minor then

Comment (Fox32) [reply .changeset/popular-nails-agree.md]: Yep, updated

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4571-bmn58uma9t6a7kwytwzrj7lfalydpfqe.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/602e3d8dfcb22d554558face

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4571-bmn58uma9t6a7kwytwzrj7lfalydpfqe.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/602e3d8dfcb22d554558face

**Comment by Fox32** [comment]
I guess the broken build is not related to my changes?

**Comment by freben** [comment]
Nah, let me see if we can look at that

**Review by freben — COMMENTED** [review]
Nice! Let's just address the type/subtitle thing. If you make it backwards compatible and deprecate the old one, you can demote the core change to `patch` again. Then let's :shipit:

**Comment by freben** [comment]
Oh and also - rebasing on master should fix CI

**Comment by regicsolutions** [comment]
Looks great @Fox32 Another idea that came to mind for the explore plugin was to leverage it to publish Product Standards and Patterns, not sure how your team is managing that now but we are using a mixture of SharePoint lists and word documents...

**Comment by Fox32** [comment]
> Looks great @Fox32 Another idea that came to mind for the explore plugin was to leverage it to publish Product Standards and Patterns, not sure how your team is managing that now but we are using a mixture of SharePoint lists and word documents...

You could actually use the tools section in the explore plugin for this. As an alternative: techdocs FTW

File: .changeset/popular-nails-agree.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+---
+
+Rename `type` of `ItemCard` to `subtitle` and allow to pass react nodes.
```
[the remark this answers is not in the document]
Comment (Fox32) [reply .changeset/popular-nails-agree.md:5]: > Subtitle drawn above a main title? Unheard of! :D

I would say that it's quite a common pattern, but actually I can come up with an example right now 😆

**Comment by Fox32** [comment]
Now that you can color links, I wonder if this a pattern that we want to use in other places too, like here: 

image

File: .changeset/popular-nails-agree.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Fox32) [reply .changeset/popular-nails-agree.md]: Deprecated and back to patch

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4571-bmn58uma9t6a7kwytwzrj7lfalydpfqe.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/602e3d8dfcb22d554558face

**Review by freben — APPROVED** [review]

**Comment by freben** [comment]
Nice!

**Pull request merged**